### ISSUE TITLE
Error when toggling off (ZT)

### ISSUE THREAD

**Issue by sQVe** [kamykn/spelunker.vim#17]
Hey :wave:,

I'm consistently getting errors when trying to toggle off `spelunker.vim` with `ZT`. I'm getting the following error:

```shell
Error detected while processing function spelunker#toggle[1]..spelunker#toggle#toggle[5]..spelunker#matches#delete_matches:
line    5:
E735: Can only compare Dictionary with Dictionary
E15: Invalid expression: l:delete_match_id > 0
```

**Comment by kamykn** [comment]
Hi, @sQVe .
I'm sorry, this is a bug that was added when another bug was fixed.
The corresponding bug was fixed in b6c797a and abebcf9.
Please update spelunker.vim.

**Comment by sQVe** [comment]
Works as expected now @kamykn. Thank you for the quick fix. :pray:
